stylesheets/layout/_footer.scss

```diff
@@ -23,7 +23,7 @@ h3 {
     margin-bottom: 25px;
     padding-left: 15px;
   }
-  button{
+  .yellow-button{
```

Comment:
Are you sure this is the correct class name? What happens if the colors will be changed =)